polymorphic.ts

```diff
@@ -204,7 +204,7 @@ function resolvePolymorphicSwitch(
 /** Uses either the polymorphic resolver or the polymorphic switch resolver to determine the
  * appropriate class, then deserialize the input using Serializable#fromJSON, returning the result
  */
-export function polymorphicClassFromJSON<const T extends Serializable>(
```

Comment:
to be fair, const in a generic kinda feels funky, not surprised the compiler doesn't like it.

What I am suprised about is it works on github, but not locally.

Reply:
not sure it would work as well as this would enforce[ runtime generic member types ](https://xebia.com/blog/typescript-5-0-and-the-new-const-modifier-on-type-parameters/)